### ISSUE TITLE
region based raid notify rules triggering outside their regions (circle vs square regions)

### ISSUE THREAD

**Issue by wdbrs** [PokeNavBot/Issue-Tracker#629]
**Describe the bug**
I've got region based raid notify rules triggering outside their regions.  I double checked all the coordinates and radius settings under $subs, they're correct. I ran $gi on the gyms in question, they're correct and known locations. Repeatable via screenshot and commands both. Discussed on Discord, IVPips said, "For some reason I use the bounding box of the circles not the actual distance."  Making a ticket here on request

**To Reproduce**
1. Choose a gym in your server that Pokenav knows, measure a line 10 km NE, get coordinates of the location at the other end
2. Set a notify rule centered at that location, with a 9 km radius on the circle
3. Report a raid at the chosen gym
4. Region rule will trigger, since 10 km should be within the bounding box, though outside of the circle

**Expected behavior**
The guide at https://docs.pokenavbot.com/moderation.html#notifications reads
```
$create notify-rule role-name "tier: 5" "region: lat, long, 2"
- Mentions role-name when a tier 5 raid occurs within 2 km of (lat, long)
```
This means to me a circle with a given radius

**Screenshots**
[regions map](https://cdn.discordapp.com/attachments/428184932884545546/821566267986280498/notify_rule_errors.jpg) 
[erroneous report](https://cdn.discordapp.com/attachments/428184932884545546/821567387047559199/Screenshot_20210316_185548.jpg)

**Mobile OS (if specific to your device):**
NA

**Additional context**
Squares instead of circles could work, but the docs should say as much.  Or maybe both types could be supported.  There's a nice tool for planning out circles at https://www.mapdevelopers.com/draw-circle-tool.php, perhaps it should be suggested to people so they can better visualize it and don't have gaps in their gym coverage.  That could help alleviate concerns about people missing gyms, if that was a consideration when making the area larger than specified.   If it would be useful to others, to make the setup process easier, I made a Google spreadsheet to make commands for me, I just had to map the circles, then copy coordinates and radius to the spreadsheet. I'm happy to share it, check it out [here](https://docs.google.com/spreadsheets/d/1Z5vjIu0psNSsw07odrs5G3rg93HjrSog-qRcZ_v8ykw/edit?usp=sharing)

**Comment by dandesousa** [comment]
Yes currently it is a square not a circle, I will clarify the docs on this in the short term
